acra/configuring-maintaining/storage-and-data-model-implications/_index.md

```diff
@@ -2,3 +2,79 @@
 title: Storage and data model implications
 bookCollapseSection: true
 ---
+
+# Data storage
+
+Based on which Acra components you use, the requirements may differ.
+
+For example, you will most likely want to use AcraServer.
+In this case, you are limited by supported DBMS, which are MySQL, MariaDB, PostgreSQL.
```

Comment:
limited by DBMS that uses [MySQL](https://dev.mysql.com/doc/internals/en/client-server-protocol.html) and [PostgreSQL](https://www.postgresql.org/docs/current/protocol.html) wire protocols. But we should note that it regularly  tested only on postgresql/mysql/mariadb